tests/rhosak/kas_with_instance.spec.ts

```diff
@@ -184,6 +188,38 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await deleteKafkaTopic(page, testTopicName);
 });
 
+// test_4kafka.py test_kafka_topics_opened
+test('test kafka topics opened', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await page.locator('button[aria-label="Topics"]').click();
+  await expect(page.locator('h2', { hasText: 'No topics' })).toBeVisible();
+  await expect(page.locator('button', { hasText: 'Create topic' })).toBeVisible();
+});
+
+// test_4kafka.py test_kafka_topic_check_does_not_exist
+test('test kafka topic check does not exist', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await expect(page.locator('h1', { hasText: `${testInstanceName}` })).toHaveCount(1);
+  await page.locator('button[aria-label="Topics"]').click();
+  // expecting not to find topic row
+  await expect(page.getByText(testTopicName)).toBeHidden();
+});
+
+// test_4kafka.py test_kafka_try_create_topic_with_same_name
+test('test kafka try create topic with same name', async ({ page }) => {
+  await waitForKafkaReady(page, testInstanceName);
+  await page.locator('a', { hasText: `${testInstanceName}` }).click();
+  await page.locator('button[aria-label="Topics"]').click();
+  await createKafkaTopic(page, testTopicName);
+  await expect(page.locator('tr', { hasText: `${testTopicName}` })).toHaveCount(1);
+  await page.locator('button', { hasText: 'Create topic' }).click();
+  await page.getByPlaceholder('Enter topic name').fill(testTopicName);
+  await page.locator('button', { hasText: 'Next' }).click();
+  await expect(page.getByText(`${testTopicName}` + ' already exists. Try a different name')).toBeVisible();
+});
```

Comment:
> If you add more tests into `tests/rhosak/kas_with_instance.spec.ts` that create a topic with the name `testTopicName`, you also should add a cleanup to either `beforeEach` or `afterEach`. Very likely the test after `'test kafka try create topic with same name'` will fail to create topic because there will be a leftover from that test of yours.

> @mmusil Isn't that covered on JIRA https://issues.redhat.com/browse/MGDSTRM-10363 ?

@asanzgom IMO No, this should be your job as part of this PR. You have added a test, that could leave some leftovers if it fails and break the following test, so you should add a cleanup here too.

Reply:
added beforeEach() to cleanup Topic Can we resolve this conversation @mmusil ?

Reply:
Can we resolve this conversation @Frawless @mmusil ?